torch/nn/modules/flatten.py

```diff
@@ -95,14 +95,22 @@ class Unflatten(Module):
 
     __constants__ = ['dim', 'unflattened_size']
     dim: Union[int, str]
-    unflattened_size: Union[Size, NamedShape]
+    unflattened_size: Union[Size, List, NamedShape]
 
-    def __init__(self, dim: Union[int, str], unflattened_size: Union[Size, NamedShape]) -> None:
+    def __init__(self, dim: Union[int, str], unflattened_size: Union[Size, List, NamedShape]) -> None:
         super(Unflatten, self).__init__()
 
         if isinstance(dim, int):
+            if (isinstance(unflattened_size, list)):
+                unflattened_size = tuple(unflattened_size)
```

Comment:
I wonder if it's actually important that unflattened_size be a tuple and not a list.

Reply:
Yes. It only accepted tuple as the unflattened_size.

Reply:
Underlying function `torch.Tensor.unflatten` accepts list, so it should be possible to have `self.unflattened_size` attribute as a list:
```
In [6]: a=torch.randn(10,10)                                                                                                                                                                    

In [7]: a.unflatten(1, [2,5]).size()                                                                                                                                                            
Out[7]: torch.Size([10, 2, 5])
```

Reply:
oh. I see what you mean.